src/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* global document */
```

Comment:
Wasn't this in the rc file before? I think it would be safe to allow all browser globals since that's where the app will run. We could use [`env: browser`](https://eslint.org/docs/user-guide/configuring#specifying-environments), too.

Reply:
forgot about the `env` config... thanks!